README.md

```diff
@@ -283,3 +283,18 @@ if err != nil {
 }
 <-doneC
 ```
+
+#### Setting Server Time
+
+Your system time may be incorrect and you may use following function to set the time offset based off Binance Server Time:
+
+```golang
+// use the client future for Futures
+client.NewSetServerTimeService().Do(context.Background())
```

Comment:
Hi @Krisa , I think there is no need to use `NewSetServerTimeService` to set `TimeOffset`, we can directly initialize it by `client.TimeOffset = 123`

Reply:
@adshao . Absolutely, it's of course possible but I also think that it's a convenient method for many user to set the `TimeOffset` correctly. For example shall it be `currentTimestamp() - serverTime` or ... `serverTime - currentTimestamp()`. Well, it's the former, but it's not obvious at first. It's just few lines of code which I think makes life easier.

Reply:
Usually we use the style `NewXxxService.Do(ctx)` to call a remote request, it will confuse users that this method will call an API and set time offset in binance server, but actually it is just a local setting.

Reply:
@adshao - I can rename that, what would be your suggestion? `client.SetServerTime().Do(context.Background())` ?

Reply:
Refer to [https://golang.org/pkg/context/](https://golang.org/pkg/context/): 
```Package context defines the Context type, which carries deadlines, cancellation signals, and other request-scoped values across API boundaries and between processes.```
So there is a little confused to use context to set a local value, I suggest just using `client.TimeOffset`. To make it more understandable, we add comment to the code and README,  is it ok?

Reply:
The above was just a suggestion. `context` may, or may not, make sense, but I was not really discussing this. My point was rather to have a method to set this `TimeOffset`. It's for _me_ ok to not have one, but it's really error prone and at best will take time to figure out what to do, while a method, whatever its name and its arguments, could be a no-brainer...

Reply:
Sorry, I did not realize you get the current server time from binance server: https://github.com/adshao/go-binance/pull/128/files#diff-00e2923dc8514cedb6f5e824232a81c2R52, I think the solution you provided is great.

Reply:
Cool 👍 Wanna rename this method to something else or is it good to go?

Reply:
No need to rename, it's fine, thanks.